auth/jwt.go

```diff
@@ -74,6 +77,18 @@ func GetAccountID(ctx context.Context, keyfunc jwt.Keyfunc) (string, error) {
 	return "", errMissingField
 }
 
+// GetCompartmentID gets the JWT from a context and returns the CompartmentID field
+func GetCompartmentID(ctx context.Context, keyfunc jwt.Keyfunc) (string, error) {
+	var val string
+	var err error
+	val, err = GetJWTField(ctx, MultiCompartmentField, keyfunc)
+	if err == nil || err == errMissingField {
+		return val, nil
+	} else {
+		return val, err
+	}
```

Comment:
This does the same thing. Guess I'm fine with ignoring missing compartment errors.
```suggestion
	val, err := GetJWTField(ctx, MultiCompartmentField, keyfunc)
	if err == errMissingField {
		return "", nil
	}
	return val, err
```